pkg/odo/cli/config/view.go

```diff
@@ -104,5 +112,8 @@ func NewCmdView(name, fullName string) *cobra.Command {
 			genericclioptions.GenericRun(o, cmd, args)
 		},
 	}
+
+	configurationViewCmd.Flags().StringVar(&o.componentContext, "context", "", "Use context to indicate the path where the component settings need to be saved and this directory should contain component source for local and binary components")
```

Comment:
Text description is way too long. Maybe use the current text that's being used for `odo push` and `odo create` from PR #1360 ?

Reply:
most of the commands will have `--context` flag. 
We should abstract this away, similarly how we used to do `AddProjectFlag`, to make sure that the description and definition of this flag is the same everywhere.

Reply:
https://github.com/openshift/odo/pull/1527#issuecomment-479464151

Reply:
You should now be able to use genericclioptions.AddContextFlag

Reply:
resolved